hello-world-python/LICENSE.txt

```diff
@@ -0,0 +1,174 @@
+                                 Apache License
```

Comment:
You can delete this file. This is a duplication of the LICENSE present in the root directory.

Reply:
ok done